src/Sylius/Bundle/CoreBundle/Validator/Constraints/ProductVariantCodeExistsValidator.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\CoreBundle\Validator\Constraints;
```

Comment:
I'm wondering maybe we should put it in the ProductBundle, as this validator is pretty coupled with product itself, without a need of knowing about the rest of the system.